Project.toml

```diff
@@ -9,6 +9,7 @@ Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 MsgPack = "99f44e22-a591-53d1-9472-aa23ef4bd671"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 TranscodingStreams = "3bb67fe8-82b1-5028-8e26-92a6c54297fa"
+URIParser = "30578b45-9adc-5946-b283-645ec420af67"
```

Comment:
URIParser.jl is currently un(der)maintained; the most actively maintained URI parser is a separate implementation in HTTP.jl. We might want to consider using that instead.

Reply:
@ararslan is there any julia package out there that you *haven't* contributed to?  I see you on both of these..